docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -198,26 +173,23 @@ First, the `project.json` file should look something like this:
                 "System.Threading.Tasks":""
             }
         },
-        "dotnet55":{
+        "netstandard1.5":{
             "dependencies": {
-                "System.Runtime":"4.0.0-rc1-*",
-                "System.Net.Http": "4.0.1-beta-23409",
-                "System.Text.RegularExpressions": "4.0.11-beta-23409",
-                "System.Threading.Tasks": "4.0.11-beta-23409"
+                "NETStandard.Library":"1.5.0-rc2-final",
             }
         }
     }
 }
 
 ```
 
-Note that .NET Framework assemblies being used are explicitly referenced in the `net40` and `net452` target, and NuGet references are also explicitly listed in the `dotnet55` target.  This is required when cross-compiling with `#if` directives.
+Note that .NET Framework assemblies being used are explicitly referenced in the `net40` and `net452` target, and NuGet references are also explicitly listed in the `netstandard1.5` target.  This is required when cross-compiling with `#if` directives.
 
```

Comment:
'are explicitly referenced' -> 'need to be referenced explicitly'

How does this paragraph relate to #if directives?